bin/dbsync

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+var path   = require('path');
+var fs     = require('fs');
+var coffee = require('coffee-script');
+var src    = path.join(path.dirname(fs.realpathSync(__filename)), '../src');
+
```

Comment:
You're not aware of the others because I've kept my mouth shut for the most part.  ;-)  My opinions depart from the standard coffeescript style guide, and yours seem to line up with it, so I'm not vocal about asking you to change.

Regarding how it would run, I see no reason to include gulp / a build setup just to run lint.  If contribution code style becomes an issue, we could include some contributor instructions about matching existing style, or if we want to take it further and use coffeelint then we can put it in as a devDependency and put a 'lint' script in package.json -- much better to allow it to run as a development task rather than lint the code every single time the app runs.